workflows/test/transaction-monitor-workflow.test.js

```diff
@@ -0,0 +1,362 @@
+/** @import {WorkflowStep} from 'cloudflare:workers' */
+import { env } from 'cloudflare:test'
+import { describe, it, expect, vi, afterEach } from 'vitest'
+import { TransactionMonitorWorkflow } from '../lib/transaction-monitor-workflow.js'
+
+vi.mock('../lib/chain.js', () => ({
+  getChainClient: vi.fn(),
+}))
+
+describe('TransactionMonitorWorkflow', () => {
+  /** @type {WorkflowStep} */
+  const mockStep = { do: vi.fn() }
+  /** @type {{ send: (message: any) => Promise<void> }} */
```

Comment:
Please avoid `any`. You can use `unknown` to allow any message type.


```suggestion
  /** @type {{ send: (message: unknown) => Promise<void> }} */
```